src/lib.rs

```diff
@@ -61,40 +59,50 @@ pub mod swc {
         };
 
         GLOBALS.set(&Default::default(), || {
-            let input = StringInput::new(source_text, Default::default(), Default::default());
-            let mut program = Parser::new(syntax, input, Some(&comments))
-                .parse_program()
-                .unwrap();
-
-            let top_level_mark = Mark::new();
-            let unresolved_mark = Mark::new();
+            try_with_handler(cm.clone(), Default::default(), |_handler| {
```

Comment:
thread 'main' panicked at /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/scoped-tls-1.0.1/src/lib.rs:168:9:
  cannot access a scoped thread local variable without calling `set` first

Reply:
Did you get the error message from the debug build?

Reply:
CI: https://github.com/oxc-project/bench-javascript-transformer-written-in-rust/actions/runs/11973863599/job/33383679558

It's in release build https://github.com/oxc-project/bench-javascript-transformer-written-in-rust/actions/runs/11973863599/job/33383679558

Reply:
I think you need to configure https://rustdoc.swc.rs/swc_core/ecma/transforms/base/helpers/static.HELPERS.html